packages/operator/controllers/modeldeployment_controller.go

```diff
@@ -633,6 +634,48 @@ func (r *ModelDeploymentReconciler) cleanupOldRevisions(
 	return nil
 }
 
+func getCMPolicyName(modelDeploymentCR *odahuflowv1alpha1.ModelDeployment) string {
+	return modelDeploymentCR.Name + "-" + cmPolicySuffix
+}
+
+func (r *ModelDeploymentReconciler) reconcilePolicyCM(log logr.Logger,
+	modelDeploymentCR *odahuflowv1alpha1.ModelDeployment) error {
+
+	rn := modelDeploymentCR.Spec.RoleName
+	if rn == nil {
+		log.Info(".Spec.RoleName is nil. Skip creating custom policies for model")
+		return nil
+	}
+
+	policies, err := deployment.ReadDefaultPoliciesAndRender(*rn)
```

Comment:
I don't know how lightweight or heavy this operations is (not familiar with `bindata`), but maybe we could optimize it a bit if we check for existing config map **before** we build new one from templates. Anyway this is really minor and up to you.

Reply:
This operation is very cheap (reading bytes from memory buffer) so I believe that this optimization will have almost no effect